### ISSUE TITLE
Create an RC testing matrix for a11y features

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#830]
To efficiently test accessibility features we need a test matrix for the desired platforms and and features that should be included/verified for publication.

@phet-steele had some good ideas for what this might look like, and thought that it could look similar to the phet-io testing matrices since we will have platform specific features to test.

@phet-steele would you mind drafting a testing matrix?  Assigning @ariel-phet and @emily-phet so they are aware of this request.

**Comment by ariel-phet** [comment]
Needs to be high priority given #831

**Comment by phet-steele** [comment]
@emily-phet @jessegreenberg, the rc testing matrix on steroids: https://docs.google.com/spreadsheets/d/1o4I2qSap0AJbyW2lLM3S1ERvEFYawb-7_Et0Sjb_VRs/edit#gid=2

Did I miss anything? You won't be able to edit anything.

**Comment by emily-phet** [comment]
@phet-steele This looks correct to me (regarding screen readers). Regarding keyboard navigation, there are some scenarios where this is not needed, so it might be useful to grey out those cells in the Keyboard Navigation column. For example, no keyboard nav testing is needed for iPads, which (I believe) rely on bluetooth keyboards which don't work well for keyboard nav.

**Comment by phet-steele** [comment]
> For example, no keyboard nav testing is needed for iPads

@emily-phet that is the purpose of the "/" in those cells. Those cells also cannot be edited, so you can't test them if you wanted to.

**Comment by ariel-phet** [comment]
@phet-steele 2 comments (and this may apply to our main testing matrix as well)

1. I think we can remove iPad Air 2 iOS 9.3.5 - at this point I think we can safely assume that on devices that allow iOS 10 they have been updated. And if a user has a device that can be upgraded and has not yet been, that would be our first step in trouble shooting. 

2. We might want to check with Oliver, but I am inclined to remove Windows 8.1. People are mainly on 7 or 10, 8.1 is pretty rare.

I am sure about (1) but we should get Oliver to look at google analytics regarding (2) 

@jessegreenberg does voiceover and touch not work at all on an iPad with our scheme? Would it be worth at least seeing how it does if there is any compatibility?

**Comment by ariel-phet** [comment]
@oliver-phet reports Win 8.1 represents roughly 5.4% of our web traffic (2017 to date), and 10.3% of our Windows users.  So we will continue to test Win 8.1

**Comment by jessegreenberg** [comment]
> @jessegreenberg does voiceover and touch not work at all on an iPad with our scheme?

For now, accessibility is disabled on that platform.  Mobile VoiceOver has partial support, everything can be read with mobile VO, but interaction is very difficult.  VoiceOver uses CSS to know where the user's fingers should be to interact with something, and we use CSS makes everything invisible/out of the way.  We will need to consider workarounds for this problem.

**Comment by jessegreenberg** [comment]
@phet-steele JAWS and NVDA both have version numbers, can we add a column to document this?

**Comment by jessegreenberg** [comment]
@phet-steele recommended that the AT version be added in the row entry for the platform.  That works for me!

**Comment by jessegreenberg** [comment]
@phet-steele this testing matrix is basically sort of testing matrix with more stuff.  Instead of having a new spreadsheet document for each sim, should this matrix be added as a new sheet in the existing sim spreadsheet? Or does it make most sense to have them separate?

**Comment by phet-steele** [comment]
> should this matrix be added as a new sheet in the existing sim spreadsheet?

@jessegreenberg I was thinking this to myself yesterday, but couldn't decide if it was the right thing to do. If we merge the two, how would you purpose documenting that 1.3.0-rc.1 was an a11y test? Or does that even matter? If you are on that sheet for that rc version, it would be clear. But if you were on a different sheet.....should the sheet be named "1.3.0-rc.1 (a11y)"? Am I overthinking this like usual?

**Comment by jessegreenberg** [comment]
@phet-steele I think that in the long term future, we will not want to separate at all. Eventually, all sims will have these features and this matrix will replace the "normal" testing matrix.

From that perspective, maybe adding "(a11y)" to the sheet name makes sense as we transition, and we can begin to drop it as a11y becomes a standard in all sims?

**Comment by phet-steele** [comment]
> From that perspective, maybe adding "(a11y)" to the sheet name makes sense as we transition

Done here https://docs.google.com/spreadsheets/d/1weqPMQfVBHX_IDtLTcu-EyrFamRaH7xtjFkdELS2Tfg/edit#gid=853935431

We will continue this moving forward. The a11y template will remain.
